server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -99,16 +100,17 @@ public Builder threadId(String threadId) {
             return this;
         }
 
+        @JsonIgnore
         public Builder mailboxId(MailboxId mailboxId) {
-            return this.mailboxIds(ImmutableList.of(mailboxId));
+            return this.fluentMailboxIds(mailboxId);
         }
 
         @JsonIgnore
-        public Builder mailboxIds(MailboxId... mailboxIds) {
+        public Builder fluentMailboxIds(MailboxId... mailboxIds) {
```

Comment:
Just to say it, a different method name is required, otherwise Jackson fails. Hence, no mailboxIds...